ci/docker/centos7

```diff
@@ -18,7 +18,7 @@ RUN yum install -y \
 
 FROM yum AS libssh2
 RUN cd /tmp && \
-    curl https://libssh2.org/download/libssh2-1.8.0.tar.gz | tar -xz && \
+    curl https://www.libssh2.org/download/libssh2-1.8.0.tar.gz | tar -xz && \
```

Comment:
```suggestion
    curl --location https://www.libssh2.org/download/libssh2-1.8.0.tar.gz | tar -xz && \
```

might still be good to throw in the flag to let curl know to follow redirects in case the libssh2 folks want to move stuff around.

Reply:
👍  Looks like the other Dockerfiles already did this, so I just unified their arguments across the board.